tests/cmdline/commands/test_devel.py

```diff
@@ -15,6 +15,7 @@
 from aiida.orm import Node, ProcessNode, QueryBuilder
 
 
+@pytest.mark.requires_pgsql
```

Comment:
Huh, how does this one fail for SQLite? Is that SQL PSQL specific? Or is it for another reason?

Reply:
`devel_run_sql` is currently guarded by this assert

```python
assert get_profile().storage_backend == 'core.psql_dos'
```
If I get rid of it I hit this exception

```python
E      File "/home/hollas/atmospec/aiida-core/src/aiida/cmdline/commands/cmd_devel.py", line 115, in devel_run_sql
E          with create_sqlalchemy_engine(get_profile().storage_config).connect() as connection:
E                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E      File "/home/hollas/atmospec/aiida-core/src/aiida/storage/psql_dos/utils.py", line 41, in create_sqlalchemy_engine
E         hostname = config['database_hostname'] or ''
E                                ~~~~~~^^^^^^^^^^^^^^^^^^^^^
E         KeyError: 'database_hostname'
```

So making this work is beyond this PR.